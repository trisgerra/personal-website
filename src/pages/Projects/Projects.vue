<template>
  <div class="projects">
    Projects
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component({
  components: {
  },
})
export default class Projects extends Vue {}
</script>
