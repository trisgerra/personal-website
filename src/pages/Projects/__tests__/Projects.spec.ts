import { Wrapper } from '@vue/test-utils';
import Projects from '../Projects.vue';
import createWrapper from '@/utils/CreateWrapper';

describe('Projects', () => {
  it('should render as expected', () => {
    const wrapper: Wrapper<Projects> = createWrapper(Projects);
    expect(wrapper.element).toMatchSnapshot();
  });
});
