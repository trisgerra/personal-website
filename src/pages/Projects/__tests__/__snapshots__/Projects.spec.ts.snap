// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects should render as expected 1`] = `
<div
  class="projects"
>
  
  Projects

</div>
`;
